src/solvers/dg_p4est/containers.jl

```diff
@@ -41,9 +41,9 @@ Base.eltype(::ElementContainerP4est{NDIMS, RealT, uEltype}) where {NDIMS, RealT,
 
 
 struct InterfaceContainerP4est{NDIMS, uEltype<:Real, NDIMSP2} <: AbstractContainer
-  u::Array{uEltype, NDIMSP2}                  # [primary/secondary, variables, i, j, interface]
-  element_ids::Matrix{Int}                    # [primary/secondary, interfaces]
-  node_indices::Matrix{NTuple{NDIMS, Symbol}} # [primary/secondary, interfaces]
+  u::Array{uEltype, NDIMSP2}                  # [primary/secondary, variables, i, j, n_interface]
+  element_ids::Matrix{Int}                    # [primary/secondary, n_interface]
+  node_indices::Matrix{NTuple{NDIMS, Symbol}} # [primary/secondary, n_interface]
```

Comment:
This sounds weird to me. `n_interfaces` sounds like "number of interfaces" while `n_interface` sounds like "number of this interface".
Since we also write `i` and `j` and not `n_nodes`, I feel like this is a better description (although I forgot to change `variables` to `variable`).

Reply:
Ah, this makes sense! But I am equally confused by `n_interface` **not** meaning "number of interfaces", since that's what the `n_` prefix means virtually everywhere in Trixi. Thus, in this case I'd prefer to just use `variable` and `interface` (which also matches the iterator `eachinterface`).

Reply:
Yes, `interface` sounds a lot better.